bugbug/models/component.py

```diff
@@ -203,3 +204,45 @@ def is_meaningful(product, component):
 
     def get_feature_names(self):
         return self.extraction_pipeline.named_steps["union"].get_feature_names()
+
+    def check(self):
+        success = super().check()
+
+        # Check that the most meaningful product components stills have at
+        # least a bug in this component. If the check is failing that could
+        # means that:
+        # - A component has been renamed / removed
+        # - TODO: Complete this list
+
+        limit = 1
+
+        for product, component in self.meaningful_product_components:
+            query_data = [
+                # TODO: Do we want to match bugs in graveyard?
```

Comment:
You can remove the classification part. The components which are moved to Graveyard change their name anyway, so it should be OK.